src/Model/Behavior/FileStorageBehavior.php

```diff
@@ -36,9 +36,9 @@ class FileStorageBehavior extends Behavior
     protected FileStorage $fileStorage;
 
     /**
-     * @var \Phauthentic\Infrastructure\Storage\Processor\ProcessorInterface
+     * @var \Phauthentic\Infrastructure\Storage\Processor\ProcessorInterface|null
      */
-    protected ?ProcessorInterface $imageProcessor;
+    protected $imageProcessor;
```

Comment:
Why did you remove the type?

Reply:
We talked about this, the lazyloading breaks with types
or we need to push a null into it as you said.